python/tvm/meta_schedule/__init__.py

```diff
@@ -16,3 +16,4 @@
 # under the License.
 """Package `tvm.meta_schedule`. The meta schedule infrastructure."""
 from . import builder
+from . import tune_context
```

Comment:
Making it top-level indexable with `tvm.meta_schedule.TuneContext`

```suggestion
from .tune_context import TuneContext
```